src/theme/components/hero.ts

```diff
@@ -156,18 +156,16 @@ const primary = definePartsStyle(({ foregroundColor, isDarkText }) => ({
     backgroundSize: "cover",
     backgroundPosition: "center",
     display: "grid",
-    gridTemplateColumns: { base: "100%", md: "repeat(6, 1fr)" },
-    gridTemplateAreas: {
-      base: `"content"`,
-      md: `".  content content content content ."`,
-    },
+    gridTemplateColumns: "repeat(12, 1fr)",
+    gap: { base: "1.5rem", lg: "2rem" },
     minHeight: "352px",
     p: "l",
+    paddingInline: "1rem",
```

Comment:
Why both of these? If they are both needed, you can use `s` for `paddingInline`.